components/eam/src/dynamics/se/se_iop_intr_mod.F90

```diff
@@ -442,6 +438,10 @@ subroutine apply_iop_forcing(elem,hvcoord,hybrid,tl,n,t_before_advance,nets,nete
 
   enddo
 
+  if (iop_coriolis) then
+    call iop_apply_coriolis(elem,t1,nelemd_todo,np_todo,dt)
+  endif
+
```

Comment:
@bogensch I made this change to EAM to fix the fact that we were looping over number of elements twice for applying coriolis force. Let me know if you prefer no changes to EAM and I can revert it.

Reply:
I am fine with this fix here.  Thanks for catching this!